web/libs/core/src/pages/AccountSettings/sections/index.tsx

```diff
@@ -41,7 +41,7 @@ export const accountSettingsSections = (settings: AuthTokenSettings): SectionTyp
       },
     settings.legacy_api_tokens_enabled && {
       title: isFF(FF_AUTH_TOKENS) ? "Legacy Token" : "Access Token",
-      id: "personal-access-token",
+      id: "legacy-token",
```

Comment:
what is this change for?

Reply:
This just fixes the anchor link when you click this item in the side bar, so it goes to the legacy token section (rather than the personal access token section)